src/iOS.Extension/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
Not sure why this happened. This is how old versions of nuget defined packages. These should be paclage references in the csproj file, like all others.